src/table/utils/handle-key-press.ts

```diff
@@ -1,15 +1,30 @@
+import { stardust } from '@nebula.js/stardust';
+import React from 'react';
+
 import { updateFocus, focusSelectionToolbar, getCellElement, announceSelectionState } from './handle-accessibility';
-import { SelectionActions } from './selections-utils';
+import { SelectionActions, TSelectionActions } from './selections-utils';
 import { handleNavigateTop } from './handle-scroll';
+import { AnnounceFn, Column, ExtendedSelectionAPI, TableCell, TableLayout, TotalsPosition } from '../../types';
 
-const isCtrlShift = (evt) => evt.shiftKey && (evt.ctrlKey || evt.metaKey);
+const isCtrlShift = (evt: React.KeyboardEvent) => evt.shiftKey && (evt.ctrlKey || evt.metaKey);
 
-export const preventDefaultBehavior = (evt) => {
+export const preventDefaultBehavior = (evt: React.KeyboardEvent) => {
   evt.stopPropagation();
   evt.preventDefault();
 };
 
-export const handleTableWrapperKeyDown = ({
+interface HandleWrapperProps {
```

Comment:
why not in the types file?

Reply:
good question. I haven't worried too much where the types are at this point. My innitial thought was to put types only used in one file in that file and then put types used in multiple files in the types file.

Reply:
My thought is that it is good not to make the file larger because of defining types.

Reply:
yeah, I see that too. This file even surpasses the codeclimate threshold

Reply:
moved. I think when all files are done we would need to split up the types file in a smart way